<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>formEmail</class>
 <widget class="QDialog" name="formEmail" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>E-Posta Gönder</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="simgeler.qrc" >:/new/prefix1/icons/mail_new3.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;Kime</string>
       </property>
       <property name="buddy" >
        <cstring>lineKime</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineKime" >
       <property name="toolTip" >
        <string>Birden fazla kişiye göndermek için e-posta adreslerini virgülle ayırabilirsiniz...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>ve / veya</string>
       </property>
       <property name="buddy" >
        <cstring>comboSektor</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboSektor" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>K&amp;onu</string>
       </property>
       <property name="buddy" >
        <cstring>lineKonu</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineKonu" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="metinGonder" >
     <property name="toolTip" >
      <string/>
     </property>
     <property name="tabChangesFocus" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnGonder" >
       <property name="text" >
        <string>Gönder</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>İptal</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="simgeler.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>formEmail</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>364</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
